test/system/600-completion.bats

```diff
@@ -231,6 +238,23 @@ function _check_completion_end() {
     is "${lines[-1]}" "Completion ended with directive: ShellCompDirective$1" "Completion has wrong ShellCompDirective set"
 }
 
+# check that there no suggestions in the output
+# we could only check stdout and not stderr but this is not possible with bats
+# by default we always have to extra lines at the end
+# and then we could have other extra lines for debugging, they will always start with [Debug]
+function _check_no_suggestions() {
+    if [ ${#lines[@]} -gt 2 ]; then
+        # checking for line count is not enough since we may include additional debug output
+        # lines starting with [Debug] are allowed
```

Comment:
Missing punctuation and capitalization make this hard to read. Is this your intention?
```
# ...may include additional debug output.   <---- period
# Lines starting ...   <---- capital L
```
If that's not your intention, could you find another way to make it clearer?